roles/towerdeploy/tasks/main.yml

```diff
@@ -0,0 +1,29 @@
+---
+# tasks file for towerdeploy
+
+- name: towerdeploy | install capistrano
+  ansible.builtin.command: gem install capistrano --version "{{ cap_version }}"
+  register: gemoutput
+  changed_when: gemoutput.rc != 0
+  failed_when: gemoutput.rc !=0
+
+- name: towerdeploy | install capistrano dependencies
+  ansible.builtin.command: gem install "{{ item }}"
+  register: capgemoutput
+  changed_when: capgemoutput.rc != 0
```

Comment:
Is it possible to make this more sensitive? When I added capistrano-composer it did not reflect that there was a change.

Reply:
heya @maxkadel the `ansible.builtin.command` worked more reliably `community.general.gem` in all our tests. We you expecting output?

Reply:
I have issued this https://github.com/pulibrary/princeton_ansible/issues/3781